 Request execution failed with message: java.net.ConnectException: Connection refused: getsockopt
[2m2025-03-27T11:15:05.418+05:30[0;39m [31mERROR[0;39m [35m34040[0;39m [2m--- [demo] [beatExecutor-%d] [0;39m[36mcom.netflix.discovery.DiscoveryClient   [0;39m [2m:[0;39m DiscoveryClient_DEMO/ULW5CG3330XHT.ent.covance.com:demo - was unable to send heartbeat!

 Request execution failed with message: java.net.ConnectException: Connection refused: getsockopt
[2m2025-03-27T11:15:05.418+05:30[0;39m [31mERROR[0;39m [35m34040[0;39m [2m--- [demo] [beatExecutor-%d] [0;39m[36mcom.netflix.discovery.DiscoveryClient   [0;39m [2m:[0;39m DiscoveryClient_DEMO/ULW5CG3330XHT.ent.covance.com:demo - was unable to send heartbeat!

: Connection refused: getsockopt

[2m:[0;39m Request execution error. endpoint=DefaultEndpoint{ serviceUrl='http://localhost:8761/eureka/}, exception=java.net.ConnectException: Connection refused: getsockopt stacktrace=jakarta.ws.rs.ProcessingException: java.net.ConnectException: Connection refused: getsockopt


[2m2025-03-27T11:13:05.443+05:30[0;39m [32m INFO[0;39m [35m34040[0;39m [2m--- [demo] [foReplicator-%d] [0;39m[36mcom.netflix.discovery.DiscoveryClient   [0;39m [2m:[0;39m DiscoveryClient_DEMO/ULW5CG3330XHT.ent.covance.com:demo: registering service...
[2m2025-03-27T11:13:05.475+05:30[0;39m [32m INFO[0;39m [35m34040[0;39m [2m--- [demo] [foReplicator-%d] [0;39m[36mc.n.d.s.t.d.RedirectingEurekaHttpClient [0;39m [2m:[0;39m Request execution error. endpoint=DefaultEndpoint{ serviceUrl='http://localhost:8761/eureka/}, exception=java.net.ConnectException: Connection refused: getsockopt stacktrace=jakarta.ws.rs.ProcessingException: java.net.ConnectException: Connection refused: getsockopt
	at org.glassfish.jersey.client.internal.HttpUrlConnector.apply(HttpUrlConnector.java:288)


[2m2025-03-27T11:13:35.520+05:30[0;39m [33m WARN[0;39m [35m34040[0;39m [2m--- [demo] [foReplicator-%d] [0;39m[36mc.n.discovery.InstanceInfoReplicator    [0;39m [2m:[0;39m There was a problem with the instance info replicator

Request execution failed with message: java.net.ConnectException: Connection refused: getsockopt
[2m2025-03-27T11:13:05.286+05:30[0;39m [31mERROR[0;39m [35m34040[0;39m [2m--- [demo] [beatExecutor-%d] [0;39m[36mcom.netflix.discovery.DiscoveryClient   [0;39m [2m:[0;39m DiscoveryClient_DEMO/ULW5CG3330XHT.ent.covance.com:demo - was unable to send heartbeat!

this error is coming while running the aplicatino fix this 
